README.md

```diff
@@ -52,16 +52,19 @@ into version control.
 ### End-to-end Tests
 
 The end-to-end test suite is run with `stack test :e2e`. It requires you to
-start up an IPFS test network with
+start up an IPFS test network and the Raicle daemon.
 
     docker-compose -f test/docker-compose.yaml up -d ipfs-test-network
+    RAD_IPFS_API_URL=http://localhost:19301 stack exec -- radicle-daemon --filePrefix test
```

Comment:
Isn't `--file-prefix` the more common option style?

Reply:
Yes, but the current implementation requires `--filePrefix`. I’ll change that in an upcoming commit.